src/source/feeds.rst

```diff
@@ -31,6 +81,680 @@ XML file.
+
+======= ========================================================================================================================
+Example .. code-block:: html 
+
+            <ad:media>
+                <ad:image url="https://images.pexels.com/photos/62289/yemen-chameleon-chamaeleo-calyptratus-chameleon-reptile-62289.jpeg"/>
+                <ad:image url="https://images.pexels.com/photos/47547/squirrel-animal-cute-rodents-47547.jpeg?auto=compress&amp;cs=tinysrgb&amp;w=1260&amp;h=750&amp;dpr=2"/>
+            <ad:media/>
+======= ========================================================================================================================
+
+.. index:: attributes
+.. _feed_attr:
+
+attributes
+""""""""""
+
+Optional collection of product :ref:`user_defined_attributes` (category-dependent), that can be used to influence the ad relevance. 
+
+======= =========================================================
+Example .. code-block:: html 
+
+            <ad:attributes>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>nl</ad:attributeLocale>
+                    <ad:attributeLabel>Kleur</ad:attributeLabel>
+                    <ad:attributeValue>Rood</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>en</ad:attributeLocale>
+                    <ad:attributeLabel>Color</ad:attributeLabel>
+                    <ad:attributeValue>Red</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>Model</ad:attributeName>
+                    <ad:attributeValue>Slim</ad:attributeValue>
+                    <ad:attributeValue>Pro</ad:attributeValue>
+                </ad:attribute>
+            </ad:attributes>
+
+======= =========================================================
+
+.. index:: budgetDetails
+.. _feed_budget:
+
+budgetDetails
+"""""""""""""
+
+Section with budget details
+
+============= ========================================== ========
+Name          Description                                Required
+============= ========================================== ========
+autobid       use auto bidding option true/false         No
+cpc           CPC for the given ad in cents              No
+totalBudget   total budget for the given ad in cents     No
+dailyBudget   daily budget for the given ad in cents     No
+============= ========================================== ========
+
+The minimum and maximum values for the total budget depend on the category. 
+
+When this value of the daily budget is reached the ad will be offline for the rest of the day, and re-activated at the beginning of the following day. 
+The minimum value depends on the category.
```

Comment:
repetition of the above? why not change the statement in L368 for both total AND daily? was there a difference in treatment?

Reply:
as for the removed part

> Maximum value cannot be higher than the total budget.

my wild guess is that the intention was to say that the maximum value cannot be _lower_ than the current total spent, otherwise it will result in pausing the ad (in sellside I believe we do this validation - in feeds not sure but easy to check). @twessling-icas @pmongh what do you think?

Reply:
because (unlike console) it's not presented as a "top up" but just absolute maximum value

Reply:
please help rewriting this section

Reply:
`The minimum value of the total budget cannot be lower than the current spent for that ad` is what I would add, if anything at all (I find it useful to not surprise users if we invalidate) but I'm curious what Ton thinks

Reply:
I really hope _you_ understand the point, cause otherwise slim chance the sellers will :P

Reply:
updated